src/builtins.ts

```diff
@@ -519,6 +520,8 @@ function isinstance(t: Thread, f: IPy_FrameObj, args: IPy_Object[], kwargs: Py_D
   var obj = args[0],
     cls = args[1];
 
+  if (obj === undefined || cls === undefined)
```

Comment:
Curly brackets here would be nice.

Also, it would be good to have a comment here explaining why `obj` and/or `cls` could be undefined at this point. That is, is this expected behavior for normal programs, or this an indicator of something strange going on in the interpreter?